rosserial_tivac/src/rosserial_tivac/make_libraries

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
```

Comment:
I would have to know if the script is called by `ros_client` cmake function `rosserial_generate_ros_lib` or by the user on the terminal to generate Energia's library.

I guess I can add an extra argument to the script, which can be assigned like this:
```
rosserial_generate_ros_lib(
  PACKAGE rosserial_tivac
  SCRIPT make_libraries.py <not_energia_argument>
)
```

The reason I divided into `ros_lib` and `ros_lib_energia` is because:
1. When I tried, Energia compiled all files that are copied into its library folders.
2. Probably cleaner maintenance.